packages/tooltip/style/index.css

```diff
@@ -15,6 +15,12 @@
 }
 
 
+.jp-Tooltip > .jp-RenderedText,
+.jp-Tooltip > .jp-RenderedText > pre {
+  overflow: visible;
```

Comment:
why is this rule needed? At first glance, this seems like a bandaid for a CSS problem elsewhere.